src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
+  >
+    <div
+      v-if="iconName"
+    >
+      <ec-icon
+        class="ec-summary-info__icon"
+        data-test="ec-summary-info__icon"
+        :class="{'tw-mr-12': iconName}"
+        :name="iconName"
+        :size="12"
+      />
+    </div>
+
+    <div class="ec-summary-info__content">
+      <div
+        v-if="title"
+        class="ec-summary-info__title"
+        data-test="ec-summary-info__title"
+      >
+        {{ title }}
+      </div>
+
+      <div
+        v-if="lineItems.length > 0 || hasSlot('default')"
+        data-test="ec-summary-info__content-lines"
+        class="ec-summary-info__content-lines"
+      >
+
+        <div
+          v-for="item in lineItems"
+          :key="item.text"
+          :class="[item.cssClasses]"
+          class="ec-summary-info__content-line-item"
+        >
+          <span
```

Comment:
https://www.figma.com/file/um1wKnapCkOe0BKkzIgXAY/Add-Benef?node-id=758%3A44993&t=Hr8qrbtfJmXIhUTh-4 regarding this we should provide title attribute here with the `item.text` value

Reply:
done, thanks